app/assets/stylesheets/app.scss

```diff
@@ -1,3 +1,3 @@
 .main-container {
-  min-height: calc(100vh - 72px - 70px);
+  min-height: calc(100vh - 72px - 35px);
```

Comment:
since there are two static values (72px and 35px) you can simplify this to be just `calc(100vh - 37px)`